src/ch4_component.cpp

```diff
@@ -98,6 +98,11 @@ void CH4Component::setData(const string &varName, const message_data &data) {
   try {
     if (varName == D_PREINDUSTRIAL_CH4) {
       H_ASSERT(data.date == Core::undefinedIndex(), "date not allowed");
+        if (data.getUnitval(U_PPBV_CH4).value(U_PPBV_CH4) != 731.41) {
+            H_LOG(logger, Logger::WARNING) << "Changing " << varName <<
+                " from default value, this is not recomended and may cause" <<
```

Comment:
Just to be my grammatically picky self: better use a semicolon here and below, not a comma. 😄